app/WorkItem.cs

```diff
@@ -1,17 +1,22 @@
+using ConsoleTables;
+using lms.models;
 using Lms;
 
-enum Field {
-    Title,
-    DueAt
-}
```

Comment:
Another case of scope creep unfortunately. I don't disagree that this could be a good change, but it is unrelated to the issues this branch is linked to. Can you revert?

Reply:
Thanks for the Feedback Emil! I will make these changes. However, i still have some confusion about Implementing Fields. Currently Field is a global enum, so i cannot create another enum for progresses. I created the extra Items class because of this. I am not sure how to implement the Field to progress. The one possible way is to just add the extra field names to this Field, which is in workItems. However, that seems a bit weird to access it from workItems.

Reply:
Hmm good point. My long-term thinking would be eventually moving WorkItem and Progress to their own namespace.

In the short-term, can you make the enum `ProgressField`. Leave the WorkItem `Field` the same for now, we can tackle that later :)